src/Networking/NexusMods.Networking.NexusWebApi/Auth/JWTToken.cs

```diff
@@ -63,7 +63,7 @@ public static Optional<EntityId> Create(IDb db, ITransaction tx, JwtTokenReply r
 
         tx.Add(entityId, AccessToken, reply.AccessToken);
         tx.Add(entityId, RefreshToken, reply.RefreshToken);
-        tx.Add(entityId, ExpiresAt, DateTimeOffset.FromUnixTimeSeconds(reply.CreatedAt).DateTime + TimeSpan.FromSeconds(reply.ExpiresIn));
+        tx.Add(entityId, ExpiresAt, DateTimeOffset.FromUnixTimeSeconds(reply.CreatedAt).UtcDateTime + TimeSpan.FromSeconds(reply.ExpiresIn));
```

Comment:
The `DateTime` conversion was a left-over from when I updated the attributes to use `DateTimeOffset`.

```suggestion
        tx.Add(entityId, ExpiresAt, DateTimeOffset.FromUnixTimeSeconds(reply.CreatedAt) + TimeSpan.FromSeconds(reply.ExpiresIn));
```